lib/server.js

```diff
@@ -75,7 +75,7 @@ class Server extends KarmaEventEmitter {
       emitter: ['value', this],
       server: ['value', this],
       watcher: ['value', watcher],
-      launcher: ['type', Launcher],
+      launcher: ['factory', Launcher.factory],
```

Comment:
Could this be an API change, requiring new major version rather then refactor?

Reply:
No, it should not. This only changes how new instance is created and consumers shouldn't normally create new launcher themselves. How launcher is injected stays exactly the same and should not require any changes in plugins.